extensions/amp-list/0.1/test/validator-amp-list.html

```diff
@@ -86,6 +86,32 @@
     src="https://data.com/articles.json?ref=CANONICAL_URL"
     auto-resize>
   <div></div>
+  <!-- Valid amp-list with load-more="auto" attribute -->
+  <amp-list width=10 height=10
```

Comment:
valid test case with just `load-more` for the default case?

Reply:
So even though I have not programmatically enforced this, I think the conclusion of the discussion on https://github.com/ampproject/amphtml/pull/19967/files#diff-4154928399fad29a55f4c6450302eef6R300 is that we should only have `auto` or `manual`, and remove the default option to avoid confusion. @CrystalFaith @andrewwatterson please keep me honest. xD

Reply:
👍